pages/common/nslookup.md

```diff
@@ -0,0 +1,27 @@
+# nslookup
+
+> Query name server(s) for various domain records.
+
+- Query your system's default name server for an IP address (A record) of the domain:
+
+`nslookup {{example.com}}`
+
+- Query non default name server (here Google DNS 8.8.8.8) for a NS record of the domain:
+
+`nslookup -type=NS {{example.com}} 8.8.8.8`
+
+- Query default name server for a reverse lookup (PTR record) of an IP address:
+
+`nslookup -type=PTR 54.240.162.118`
```

Comment:
Wrap the ip in tokens {{}}. And I would suggest generalising IP to {{ip_address}}. Since you are mentioning example.com, and not a specific domain name.